roles/parse_operator_metadata/tasks/main.yml

```diff
@@ -65,7 +65,7 @@
     register: kind_custerserviceversion_file_paths
 
   - name: "Grep paths with current_csv ClusterServiceVersion"
-    shell: "grep -l {{ current_csv }} {{ item.path }}"
+    shell: "grep -l 'name: {{ current_csv }}' {{ item.path }}"
```

Comment:
should the `name: {{ current_csv }}` handle more than just a space?

Reply:
I don't think we need to. Since we are using a grep command to identify the 
```"name: <CSV_NAME>"``` inside a YML regardless of the syntax it should be enough. 
Other possibilities maybe like 
```"name:{{ current_csv }}"``` without space would be invalid YAML syntax would fail before loading any environment. 
```"name: {{ current_csv }}               " ``` string with more spaces end will be matched by grep command. 
Further, the ClusterServiceVersion inside the file paths occurs is 
```
"name: {{ current_csv }}" 
```
or 
```
replaces: {{ current_csv }}
```
please let me know if there are any other occurrences that might be seen inside YAML files.